docs/src/dataDisplay/chip/examples/exampleDeletableChip.jsx

```diff
@@ -0,0 +1,42 @@
+import {
+    Chip,
+} from 'react-cm-ui';
+import makeStyles from 'react-cm-ui/styles/makeStyles';
+import React from 'react';
+
+const useStyles = makeStyles(({
+    spacing,
+}) => ({
+    root: {
+        display: 'flex',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
+        '& > *': {
+            margin: spacing(0.5),
+        },
+    },
+}));
+
+function ExampleDeletableChip() {
+    const classes = useStyles();
+
+    const onDelete = () => {
+        console.info('You clicked the delete icon.');
```

Comment:
Do we need to suppress ESLINT warning about `console` usage?

Reply:
`console.info` doesn't give us nag.

Reply:
Nice!  Resolving.